### PR TITLE
[OSDOCS-7777]: Port content "Creating a ROSA cluster with STS using custom KMS key" from mobb.ninja to ROSA 

### PR DESCRIPTION
**Pull request by fmcdonal**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7777

Link to docs preview:
https://65044--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by fmcdonal** [comment]
@michaelryanmcneill

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+This guide will walk you through installing ROSA (Red Hat OpenShift Service on AWS) with a customer-provided KMS key that will be used to encrypt both the root volumes of nodes as well as persistent volumes for mounted EBS claims.
+====
+== Prerequisites
+
+* link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[AWS CLI]
+* link:https://github.com/openshift/rosa/releases/[ROSA CLI] v1.1.11 or higher
+* OpenShift CLI - `rosa download openshift-client`
+
+== Prepare AWS Account for ROSA
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:34]:
```suggestion
== Prepare AWS account for ROSA
```

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+The `AWS Access Key ID` and `AWS Secret Access Key` values can be obtained by logging in to the AWS console and creating an *Access Key* in the *Security Credentials* section of the IAM dashboard for your user.
++
+. Validate your credentials:
++
+[source,terminal]
+----
+$ aws sts get-caller-identity
+----
++
+You should receive output similar to the following.
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:68]:
```suggestion
You should receive output similar to the following:
```

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+. If this is a brand new AWS account that has never had a AWS Load Balancer (ALB) installed in it, run the following:
++
+[source,terminal]
+----
+$ aws iam create-service-linked-role --aws-service-name \
+    "elasticloadbalancing.amazonaws.com"
+----
++
+. Set the AWS region you plan to deploy your cluser into. For this example, we will deploy into `us-east-2`.
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:91]:
```suggestion
. Set the AWS region you plan to deploy your cluser into. For this example, we will deploy into `us-east-2`:
```

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
++
+[source,terminal]
+----
+$ export AWS_REGION="us-east-2"
+----
+
+== Create ROSA Cluster
+
+. Make sure your ROSA CLI version is at minimum v1.1.11 or higher.
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:100]:
```suggestion
. Make sure your ROSA CLI version is at minimum v1.1.11 or higher:
```

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+If you have already installed account-roles into your aws account, you can skip this step.
+====
+
++
+[source,terminal]
+----
+$ rosa create account-roles --mode auto -y
+----
+
+. Set Environment Variables.
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:120]:
```suggestion
. Set Environment Variables:
```

File: rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+$ rosa create account-roles --mode auto -y
+----
+
+. Set Environment Variables. 
++
+[source,terminal]
+----
+$ ROSA_CLUSTER_NAME=poc-kmskey
+----
+. Using the ROSA CLI, create your cluster.
```
Comment (bhardesty) [inline rosa_tutorials/creating-a-rosa-cluster-in-sts-mode-with-custom-kms-key.adoc:126]:
```suggestion
. Using the ROSA CLI, create your cluster:
```

**Comment by fmcdonal** [comment]
@michaelryanmcneill please review at your comvenience

**Comment by mgahagan73** [comment]
I think the doc would flow better and be easier to follow if put the cluster creation step after creating the key and setting the policy for it. Customers may have an existing KMS key but it appears they would still need to set the policy on it regardless.

**Comment by mgahagan73** [comment]
We also partially duplicate this content here:
https://65076--docspreview.netlify.app/openshift-rosa/latest/rosa_tutorials/rosa-mobb-security-reference#rosa-mobb-security-reference-day-one-config-create-key

**Comment by fmcdonal** [comment]
@mgahagan73 The duplication has been tracked, so we will address this when we do complete edits on the documentation. Is this ticket be ready for merging?

**Pull request closed without merging**